vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/events/MapClickEvent.java

```diff
@@ -50,6 +65,19 @@ public Coordinate getCoordinate() {
         return coordinate;
     }
 
+    /**
+     * List of map features at the clicked position. Can be used to distinguish
+     * whether the click was on the background, or on a feature. The features
+     * are sorted by their display order, meaning the top-most feature is the
+     * first item in the list.
+     *
+     * @return the list of features at the clicked position, or an empty list
+     *         otherwise
+     */
+    public List<FeatureEventDetails> getFeatures() {
+        return features;
```

Comment:
Thinking of whether it should return an unmodifiable list or not.

Reply:
Sure, can't hurt. Done.